lib/webhook-proxy.js

```diff
@@ -1,14 +1,38 @@
 module.exports = ({url, port, path, logger}) => {
+  
   try {
-    const SmeeClient = require('smee-client')
-
-    const smee = new SmeeClient({
+    var goodUrl = true; //url is considered to be true;
+    const SmeeClient = require('smee-client');
+    const valid= ValidURL(url);
+    if(!valid)
```

Comment:
this can be changed to:

```js
if (validURL(url))
```

This way you don't need the variable.

Reply:
Ah yes. I'm sorry, my bad. Thanks for directing me. I'll rectify it.